packages/verified-fetch/src/utils/response-headers.ts

```diff
@@ -0,0 +1,53 @@
+import { CodeError } from '@libp2p/interface'
+import type { SupportedBodyTypes } from '../types.js'
+
+/**
+ * Gets the body size of a given body if it's possible to calculate it synchronously.
+ */
+function syncBodySize (body: SupportedBodyTypes): number | null {
```

Comment:
Minor nit but synchronous APIs normally have "sync" appended, not prepended.  For example the [fs sync API](https://nodejs.org/api/fs.html#synchronous-api).

```suggestion
function getBodySizeSync (body: SupportedBodyTypes): number | null {
```